src/audio/module_adapter/module/volume/volume_generic_with_peakvol.c

```diff
@@ -310,7 +310,8 @@ static void vol_s16_to_s16(struct processing_module *mod, struct input_stream_bu
 							      Q_SHIFT_BITS_32(15, VOL_QXY_Y, 15));
 				tmp = MAX(abs(x0[i]), tmp);
 			}
-			cd->peak_regs.peak_meter[j] = MAX(tmp, cd->peak_regs.peak_meter[j]);
+			cd->peak_regs.peak_meter[j] = MAX(tmp, cd->peak_regs.peak_meter[j])
```

Comment:
suggest tmp type to uint32_t instead of int, since it is for peak meter.
prefer left shift with tmp to get better code looking:

tmp <<= PEAK_16S_32C_ADJUST;
cd->peak_regs.peak_meter[j] = MAX(tmp, cd->peak_regs.peak_meter[j]);

looks better?